rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -260,12 +235,12 @@ namespace
 			mip = D3D12_FILTER_TYPE_LINEAR;
 			return;
 		case CELL_GCM_TEXTURE_CONVOLUTION_MIN:
-			LOG_WARNING(RSX, "CELL_GCM_TEXTURE_CONVOLUTION_MIN not supported, fallback to bilinear filtering");
 			min = D3D12_FILTER_TYPE_LINEAR;
 			mip = D3D12_FILTER_TYPE_POINT;
 			return;
 		}
-		unreachable("Wrong min filter");
+		unreachable("Unimplemented min filter used %x, please report this to a developer.", min_filter);
+
```

Comment:
@danilaml This is exactly what I have written, I can not understand why @tambry my talk, I was talking to @O1L